posts/hybrid-cloud-cloud-responsibly.md

```diff
@@ -0,0 +1,94 @@
+---
+authors:
+- Lane Jennison
+- Courtney Parry
+- Rob Zabriskie
+tags:
+- Hybrid
+- Cloud
+- JHipster
+- POC
+date: 2019-08-26T12:00:00.000Z
+title: "Hybric Cloud - Cloud Responsibly"
+image: https://github.com/ippontech/blog-usa/blob/master/images/2019/08/hybrid-cloud.jpg
+---
+
+In this post, we’ll describe the <a href="#concept">hybrid cloud concept</a>, the <a href="#benefits">benefits</a> it provides, and a <a href="#architecture">reference architecture</a> to implementing it.
+
+Moving from on-prem to the cloud via “forklift migration”, where legacy systems are dumped into a cloud-based host, is inefficient and ineffective. The goal should be to strategically extend into the cloud without abandoning the on-prem infrastructure. We do this by incrementally moving services to the cloud, leveraging the advantages of the cloud platform while retaining the value of on-prem where appropriate. This approach allows us to treat cloud vendors as interchangeable based upon cost/value criteria driven by the needs of the business. 
```

Comment:
Should we mention anything about use cases where the customer may have certain things they *never* want in the cloud?

Reply:
Does not "while retaining the value of on-prem where appropriate." cover that?  Or are you thinking about something else or a greater emphasis or rewording?

Reply:
Hmmm...I suppose you are right.